packages/gatsby-source-wikipedia/src/fetch.js

```diff
@@ -20,9 +20,9 @@ const fetchNodesFromSearch = ({ query, limit = 15 }) =>
     })
   )
 
-const getMetaData = name =>
+const getMetaData = name => lang =>
```

Comment:
No reason to use currying here:

```suggestion
const getMetaData = (name, lang) =>
```